### ISSUE TITLE
Feature Request: add an option to set the point which actually counts as a flyout start point

### ISSUE THREAD

**Issue by gaussandhisgun** [ModernFlyouts-Community/ModernFlyouts#873]
**Is your feature request related to a problem? Please describe:**

Yeah, so I was trying to put my volume flyout to the bottom right corner of my screen so it will be around the actual volume indicator in my tray, and if there are any music playing, like, at all, the flyout just goes over my taskbar, while technically staying at the same position. So, I should move the flyout up to actually be able to control the music, but then it stays too high when music is stopped so i should move it back to the taskbar.

**Describe the solution you'd like:**

There is like some Minecraft mods that let user choose where he wants the widget to be pinned at. Like, Better HUD allows users to choose the exact one of nine points (top, center, bottom, left, middle, right and all possibe combinations) from which the widget will render, so if I choose bottom-right and the widget enlarges, it enlarges to the top-left. I'd like to see something like this in Modern Flyouts. Like, come on, we can make the volume bar render at the bottom of flyout, but we cant make it stay in one place when doing this, that seems kinda unfair.

**Describe alternatives you've considered:**

none

**Additional context:**

![image](https://user-images.githubusercontent.com/63512006/142065530-190391e8-0fc8-49cd-8bc6-8a2719e22ddc.png)
That Minecraft mod i was mentioning.

**Comment by eProject** [comment]
You can... 
only missing a way to add X margin from the edge X (pixels and/or %), 
and maybe detect if/when taskbar hidden ... and move accordingly

Guess this was asked already > todo list

Settings > Layout > Flyout Placement Mode  > Automatic (Instead of Manual) > Bottom Right

**Comment by Cyberdroid1** [comment]
What @eProject said is correct. Plus there is a setting to change the order for the flyouts, so that the volume flyout stays at bottom and the media flyout comes to the top.

**Comment by ShankarBUS** [comment]
There is auto placement option made just for this purpose. Only thing it lacks is margin (spacing from the screen).

**Comment by gaussandhisgun** [comment]
Yeah, that's not helpful in any way, because I was actually using the "automatic" placement and it forced the flyouts to upper left corner if I did not move them myself, and if I did move them, they start to expand downwards.

**Comment by ShankarBUS** [comment]
> Yeah, that's not helpful in any way, because I was actually using the "automatic" placement and it forced the flyouts to upper left corner if I did not move them myself, and if I did move them, they start to expand downwards. 

It is working as expected. Moving the flyout switches the placement to manual. In auto placement, you have to change the alignment to suit your need.

We have 9 different alignments including bottom right.

**Comment by gaussandhisgun** [comment]
So if I use bottom right automatic alignment and do not move my flyouts, and they appear in top left, that's totally expected and normal behavior. 🤔

**Comment by ShankarBUS** [comment]
Could you explain properly?

With some screenshots or recordings maybe?

**Comment by Cyberdroid1** [comment]
@ShankarBUS the person manually moved the flyouts, which then breaks automatic position, which is supposed to be fixed with 0.10.

@gaussandhisgun you will need to reset modernflyouts via app settings, to fix this bug.

**Comment by ShankarBUS** [comment]
> @ShankarBUS the person manually moved the flyouts, which then breaks automatic position, which is supposed to be fixed with 0.10.
> 
> @gaussandhisgun you will need to reset modernflyouts via app settings, to fix this bug.

Now I remember.
